.github/workflows/conformance-aws-cni.yaml

```diff
@@ -108,15 +108,15 @@ jobs:
             src:
               - '!(test|Documentation)/**'
 
-  # When the test-me-please trigger is used, this job is skipped if the only
-  # modified files were under test/ or Documentation/.
+  # This job is skipped if the only modified files were under test/ or
+  # Documentation/.
   installation-and-connectivity:
     needs: check_changes
     if: |
       (github.event_name == 'issue_comment' && (
         startsWith(github.event.comment.body, 'ci-awscni') ||
-        (startsWith(github.event.comment.body, 'test-me-please') && (needs.check_changes.outputs.tested == 'true'))
-      )) ||
+        startsWith(github.event.comment.body, 'test-me-please')
+      ) && needs.check_changes.outputs.tested == 'true') ||
```

Comment:
If I'm reading this correctly, it's a change in behavior. `ci-awscni` now won't trigger the test if only `test/` or `Documentation/` were modified. That was done on purpose, as I'm assuming someone writing `ci-awscni` explicitly know what they are doing.

That's useful for quick tests where you only want to send a nop change and trigger the tests.

Reply:
Indeed, I didn't think this behaviour was useful so I erred on the side of simplifying the conditions for consistency. If we think it is, I can add it back in :)

Reply:
Unless it's adding too much complexity, yes please :pray:

Reply:
PTAL.

Reply:
I'm not seeing any change :detective: Was it adding too much complexity?

Reply:
Uuuh, are you sure? The changes are here: https://github.com/cilium/cilium/compare/f8baa22cbe9fb6a943f7bdfbf04dc31ae0ca349c..e44285b0bfc7c423921b07460e0a58461fecd7ec

Reply:
Uh, why are they in the documentation commit? Might be worth updating the commit title at least.

Reply:
Oh are they? I mixed up the fixup then.

Reply:
I have reworked the commits so that the changes are in the proper place.